static/js/enketo/widgets/notewidget.js

```diff
@@ -63,6 +69,24 @@ define( function( require, exports, module ) {
         /* jshint unused:false */
     };
 
+    function applyLiveLinkHtml( $el ) {
+        var html = $el.html();
+
+        html = html.replace( /\[([^\]]*)\]\(([^)]*<[^>]*\>[^)]*)\)/gm,
+                '<a class="live-link" href="#" target="_blank">$1<span class="href">$2</span></a>' );
```

Comment:
Maybe this is a stupid question, but why not just `<a class="live-link" href="$2">$1</a>` , and no jquerying needed?